site/gatsby-site/server/fields/common.ts

```diff
@@ -233,36 +173,24 @@ export const sendEmail = async ({ recipients, subject, dynamicData, templateId }
  * 
  * @throws {Error} Throws an error if an unsupported HTTP method is provided.
  */
-export const apiRequest = async ({ path, method = "GET" }: { method?: string, path: string }) => {
+export const apiRequest = async ({ path, params = {}, method = "GET" }: { method?: string, params?: Record<string, string>, path: string }) => {
```

Comment:
(nitpick: we might consider abstracting all Atlas-facing stuff to its own module, so that this file can read Atlas-agnostic...)

Reply:
Hopefully we'll drop Atlas altogether before that 😉